lib/cfg-grammar.y

```diff
@@ -871,6 +871,7 @@ block_stmt
             free($4);
             free($10);
             last_block_args = NULL;
+            cfg_lexer_enable_immediate_backtick_substition(lexer);
```

Comment:
do we also need to skip backtick substitution in the body of the block or just in arguments? this seems to apply it for the body too.